### PR TITLE
DataGrid: Update Record Paging description

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: JSDemos/Demos/DataGrid/RecordPaging/description.md [hunk 1]
```diff
@@ -1 +1,12 @@
...
+The pager is configured in the [pager](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/) object and consists of the following elements:
+
+- **Page navigator**        
+Used to switch between pages.
+
+- **Page size selector**        
+Used to change the page size. To display this element, enable the [showPageSizeSelector](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showPageSizeSelector) option. You can also define the [allowedPageSizes](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#allowedPageSizes) and specify the initial [pageSize](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/#pageSize) in the [paging](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/) object.
+
+- **Page information**      
+Displays the current page number and total record count. To display page information, enable the [showInfo](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showInfo) option. You can also customize the [infoText](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#infoText).
```
Comment (AlexJameson) [inline JSDemos/Demos/DataGrid/RecordPaging/description.md]: For now, it seems unclear what is infoText and why we should customize it. I'm not really sure, but maybe 'You can also use the [pageSize]() option to customize row and page counters'?

Comment (RomanTsukanov) [reply JSDemos/Demos/DataGrid/RecordPaging/description.md]: @AlexJameson How about this?
```suggestion
Displays the current page number and total record count. To display page information, enable the [showInfo](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showInfo) option. You can also use the [infoText](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#infoText) option to customize the information text string.
```

**Review by AlexJameson — DISMISSED** [review]

**Review by Alyar666 — DISMISSED** [review]

File: JSDemos/Demos/DataGrid/RecordPaging/description.md [hunk 2]
```diff
@@ -1 +1,12 @@
-The **DataGrid** widget can automatically split its content across multiple pages and allow end users to&nbsp;navigate records using its built-in pager. This demo illustrates how you can change pager settings by&nbsp;using both the **paging** and **pager** configuration options.
\ No newline at end of file
+The **DataGrid** automatically splits records across multiple pages. This optimizes performance as the **DataGrid** loads and renders only the current page records. Users can scroll the pages or use the pager to navigate between them. This demo illustrates the second case.
```
Comment (Abadzhev) [inline JSDemos/Demos/DataGrid/RecordPaging/description.md]:
```suggestion
The **DataGrid** splits records into multiple pages. This technique helps optimize the control's performance: the client only needs to load and render one page at a time. Users can use a scroll bar or a pager to navigate between pages. This demo shows how to enable and customize the pager.
```

File: JSDemos/Demos/DataGrid/RecordPaging/description.md [hunk 3]
```diff
@@ -1 +1,12 @@
...
+
+The pager is configured in the [pager](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/) object and consists of the following elements:
```
Comment (Abadzhev) [inline JSDemos/Demos/DataGrid/RecordPaging/description.md]:
```suggestion
The pager is configured in the [pager](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/) object and contains the following elements:
```

File: JSDemos/Demos/DataGrid/RecordPaging/description.md [hunk 4]
```diff
@@ -1 +1,12 @@
...
+
+- **Page navigator**        
+Used to switch between pages.
```
Comment (Abadzhev) [inline JSDemos/Demos/DataGrid/RecordPaging/description.md]:
```suggestion
Enables page navigation.
```

File: JSDemos/Demos/DataGrid/RecordPaging/description.md [hunk 5]
```diff
@@ -1 +1,12 @@
...
+
+- **Page size selector**        
+Used to change the page size. To display this element, enable the [showPageSizeSelector](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showPageSizeSelector) option. You can also define the [allowedPageSizes](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#allowedPageSizes) and specify the initial [pageSize](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/#pageSize) in the [paging](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/) object.
```
Comment (Abadzhev) [inline JSDemos/Demos/DataGrid/RecordPaging/description.md]:
```suggestion
Changes the page size. To display this element, enable the [showPageSizeSelector](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showPageSizeSelector) option. You can also define the [allowedPageSizes](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#allowedPageSizes) and specify the initial [pageSize](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/#pageSize) in the [paging](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/) object.
```

File: JSDemos/Demos/DataGrid/RecordPaging/description.md [hunk 6]
```diff
@@ -1 +1,12 @@
-The **DataGrid** widget can automatically split its content across multiple pages and allow end users to&nbsp;navigate records using its built-in pager. This demo illustrates how you can change pager settings by&nbsp;using both the **paging** and **pager** configuration options.
\ No newline at end of file
+The **DataGrid** splits records into multiple pages. This technique helps optimize the control's performance: the client only needs to load and render one page at a time. Users can use a scroll bar or a pager to navigate between pages. This demo shows how to enable and customize the pager.
```
Comment (albertov05) [inline JSDemos/Demos/DataGrid/RecordPaging/description.md]: _This technique helps optimize the control's performance: the client only needs to load and render one page at a time. Users can use a scroll bar or a pager to navigate between pages._

Is this a "technique", or do you mean "feature"?

File: JSDemos/Demos/DataGrid/RecordPaging/description.md [hunk 7]
```diff
@@ -1 +1,12 @@
...
+
+The pager is configured in the [pager](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/) object and contains the following elements:
+
+- **Page navigator**        
+Enables page navigation.
+
+- **Page size selector**        
+Changes the page size. To display this element, enable the [showPageSizeSelector](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showPageSizeSelector) option. You can also define the [allowedPageSizes](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#allowedPageSizes) and specify the initial [pageSize](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/#pageSize) in the [paging](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/) object.
```
Comment (albertov05) [inline JSDemos/Demos/DataGrid/RecordPaging/description.md:9]: _You can also define the [allowedPageSizes](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#allowedPageSizes) and..._

Is allowedPageSizes a method, an option, or something else? I think you should specify.

Comment (RomanTsukanov) [reply JSDemos/Demos/DataGrid/RecordPaging/description.md:9]: It is an option but its name, as well as that of `pageSize`, is used in this sentence as plain text: "You can also define the allowed page sizes and specify the initial page size."

File: JSDemos/Demos/DataGrid/RecordPaging/description.md (hunk 6 above)
Comment (RomanTsukanov) [reply JSDemos/Demos/DataGrid/RecordPaging/description.md]: I would say it is more a technique than a feature.

File: JSDemos/Demos/DataGrid/RecordPaging/description.md (hunk 7 above)
Comment (albertov05) [reply JSDemos/Demos/DataGrid/RecordPaging/description.md:9]: In that case, can we spell it out as "allowed page size" (use the singular form of "size"), like in your suggestion?
Also, we could change "pageSize" to "page size".

File: JSDemos/Demos/DataGrid/RecordPaging/description.md (hunk 6 above)
Comment (albertov05) [reply JSDemos/Demos/DataGrid/RecordPaging/description.md]:
```suggestion
The **DataGrid** splits records into multiple pages. This technique helps optimize control performance: the client only needs to load and render one page at a time. Users can use a scroll bar or a pager to navigate between pages. This demo shows how to enable and customize the pager.
```

File: JSDemos/Demos/DataGrid/RecordPaging/description.md (hunk 7 above)
Comment (RomanTsukanov) [reply JSDemos/Demos/DataGrid/RecordPaging/description.md:9]: No, because this is an option name.

Comment (albertov05) [reply JSDemos/Demos/DataGrid/RecordPaging/description.md:9]: Okay, I understand. Thank you.

**Pull request merged**